recipes/hmftools-cuppa/build.sh

```
@@ -1,12 +1,17 @@
 #!/bin/bash
 
 TGT="$PREFIX/share/$PKG_NAME-$PKG_VERSION-$PKG_BUILDNUM"
-[ -d "$TGT" ] || mkdir -p $TGT/{,chart/}
+[ -d "$TGT" ] || mkdir -p $TGT
 [ -d "${PREFIX}/bin" ] || mkdir -p "${PREFIX}/bin"
 
 cd "${SRC_DIR}"
```

Shellcheck

[warning] Use 'cd ... || exit' or 'cd ... || return' in case cd fails.

(SC2164)